.github/workflows/docker-images.yaml

```
@@ -0,0 +1,73 @@
+# Test our docker images are built with the most recent version
+name: Docker images
```

Nit: 

```suggestion
name: Test Docker images
```

To use the same naming as we use in other tests like "Test Update and Downgrade"